new/LIT7498OEgzeteyaMaryam.xml

```diff
@@ -0,0 +1,314 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7498OEgzeteyaMaryam" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ኦእግዝእትየ፡ ማርያም</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">ʾO-ʾƎgzǝʾǝtǝya Māryām </title>
+                <title xml:lang="en" corresp="#t1">O My Lady Mary</title>
+                <title xml:lang="en" xml:id="t2">ʾO-ʾƎgzǝʾǝtǝya Māryām-type stanzas of the miracles of Mary</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="GS"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine
+                    multimediale Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                        licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <listWit>
+                    <witness corresp="EMML7366"/>
+                    <witness corresp="EMML8920"/>
+                    <witness corresp="ESmac001"/>
+
+                </listWit>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p> The ʾO-ʾƎgzǝʾǝtǝya Māryām hymn stanzas,  consists 55 poetic textual collections serving as  summarizing hymnographic stanzas
+            of the Miracles of Mary. These stanzas were possibly authored by the monk
+                        <persName ref="PRS14890Waldakirstos "/>. Among them, 27 stanzas correspond
```

Comment:
There is one space too much in `"PRS14890Waldakirstos "` (here and below in the text)